APIURLS.md

```diff
@@ -0,0 +1,110 @@
+Admin Features:
+    Manage Project:
+        Projects Database:
+            localhost:8080/API/v1/project/id/1
+            [Post, Put, Get, Delete]
+        Create Project:
+            POST localhost:8080/API/v1/projects/id
+        Search Project:
+            GET localhost:8080/API/v1/projects?search=id
+        Delete Project:
+            Delete localhost:8080/API/v1/projects/id
+        List Project Table:
+            GET localhost:8080/API/v1/projects
+    User Management:
+        Users Database:
+            localhost:8080/API/v1/User/id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/Users
+        Invite Member: (send user invitation by email)
+            POST localhost:8080/API/v1/projects/id/Users/id
+        List User Table:
+            GET localhost:8080/API/v1/projects/id/Users
+        User status:
+            GET localhost:8080/API/v1/Users/id/status
+        Update User Last active:
+            GET localhost:8080/API/v1/Users/id/lastActive
+            PATCH localhost:8080/API/v1/Users/id
+        User Detailed Name Card:
+            GET localhost:8080/API/v1/Users/id/fullname
+            GET localhost:8080/API/v1/Users/id/emailaddress
+            GET localhost:8080/API/v1/Users/id/publicname
+    Task Board Management:
+        Tasks Database:
+            localhost:8080/API/v1/task/id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/tasks
+        Column Search Box:
+	        GET localhost:8080/API/v1/tasks?search={keywords}
+        Task Filter:
+            GET localhost:8080/API/v1/tasks?User=id
+        Assignees:
+            GET localhost:8080/API/v1/tasks/id/assignee
+        Task number:
+            GET localhost:8080/API/v1/columns/id/tasks/count
+        Three dot actions rename:
+            PUT localhost:8080/API/v1/tasks/id/taskname
+        Three dot actions delete:
+            DELETE localhost:8080/API/v1/tasks/id
+        Task card actions copy link:
+            GET localhost:8080/API/v1/tasks/id/link
+        Task card move position:
+            PUT localhost:8080/API/v1/tasks/id/position
+        Task card actions delete:
+            DELETE localhost:8080/API/v1/tasks/id
+        Task title

```

Comment:
Merge these 2 APIs into one API and should contain below information:
`
       Task Priority
       Task title
       Task Assignee
       Due Date
       Key (e.g. SAP-18)
`